ci/install_openebs.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+#OPENEBS="$(https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-operator.yaml)"
+#CASTEMPLATE="${https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-cas-templates-pre-alpha.yaml}"
+#OPENEBSVOLUME="${https://raw.githubusercontent.com/openebs/openebs/master/k8s/demo/pvc-standard-jiva-default.yaml}"
+
+echo "--------------------Installing openebs operator---------------------------"
+sleep 10
+kubectl create -f https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-operator.yaml
+
+for i in $(seq 1 50) ; do
+    replicas=$(kubectl get deployment -n openebs maya-apiserver -o json | jq ".status.readyReplicas")
+    if [ "$replicas" == "1" ]; then
+        break
+			else
+        echo "Waiting for Maya-apiserver to be ready"
+        sleep 10
+    fi
+done
+
+for i in $(seq 1 50) ; do
+    replicas=$(kubectl get deployment -n openebs openebs-provisioner -o json | jq ".status.readyReplicas")
+    if [ "$replicas" == "1" ]; then
+        break
+			else
+        echo "Waiting for Openebs-provisioner to be ready"
+        sleep 10
+    fi
+done
+
+kubectl get pods --all-namespaces
+
+echo "------------Deploy CAS templates configuration for Maya-apiserver---------"
+kubectl create -f https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-cas-templates-pre-alpha.yaml
```

Comment:
`openebs-cas-templates-pre-alpha.yaml` rename to `openebs-pre-release-features.yaml`

Reply:
Already changed !